default/python/AI/AIFleetMission.py

```diff
@@ -596,16 +596,19 @@ def issue_fleet_orders(self):
                             )
                             allowance = CombatRatingsAI.rating_needed(fleet_remaining_rating, fleet_min_rating)
                             debug("May release ships with total rating of %.1f", allowance)
-                            ship_ids = list(self.fleet.get_object().shipIDs)
+                            org_fleet = self.fleet.get_object().shipIDs
```

Comment:
`org_fleet` is a bit confusing name, since it collection of ships.  

`for ... in ...` and `list()` uses the same things inside, which implies that we could just skip wrapping to list. 

```python
for ship_id  in self.fleet.get_object().shipIDs:
```